packages/invoices/src/OutboundShipment/DetailView/OutboundServiceLineEdit/useServiceLineColumns.tsx

```diff
@@ -63,7 +65,7 @@ export const useServiceLineColumns = (
       label: 'label.total',
       align: ColumnAlign.Right,
       width: 75,
-      accessor: ({ rowData }) => get.serviceChargeTotal(rowData),
+      accessor: ({ rowData }) => c(get.serviceChargeTotal(rowData)),
```

Comment:
offroad - I noticed that the service item currency format was incorrect. 
( was showing 7.07777777777...  when I had $7 with a tax of 10% 🙄  )